test/unit/argument_parser/detail/format_man_test.cpp

```diff
@@ -0,0 +1,225 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include <seqan3/argument_parser/all.hpp>
+#include <seqan3/argument_parser/detail/format_man.hpp>
+
+using namespace seqan3;
+
+// Reused global variables
+int option_value{5};
```

Comment:
sorry to bug you with this, but instead of using global variables please use a test fixture:

```cpp

struct my_name : public ::testing::Test
{
     int number{1};
   // you variables go here
}

TEST_F(my_name, foo_test) // has to be TEST_F ! instead of just TEST
{
   std::cout << number; // I can use number here because it is part of the my_name test fixture
}
```

and `my_name` can just be `format_man`